regex-assembly/933100.ra

```diff
@@ -11,6 +11,7 @@
     x[^m]
     xm[^l]
     xml[^\s]
+    xml\s*[^a-z]
     xml$
```

Comment:
`xml$` produces a submatch of `xml[^\s]`. I'd remove `xml$` and add a comment to `xml[^\s]` that it matchs `xml$` implicitly.

Reply:
Are you sure that `xml$` matches `xml[^\s]`?
For me it doesn't match. Am I wrong?
https://regex101.com/r/eFMckA/1

Reply:
Oh I'm so stupid... 🤦. Of course it doesn't, sorry!